src/Database/test.js

```diff
@@ -84,7 +87,7 @@ describe('Batch writes', () => {
         }),
         m4,
```

Comment:
I suggest swapping m3 and m4, just to make sure that operations mixed between collections won't break